.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: elixir
+elixir:
+  - 1.4.1
+cache:
+  directories:
+    - _build
+    - deps
+addons:
+  postgresql: '9.4'
+env:
+  - MIX_ENV=test
+before_script:
+  - mix do ecto.create
```

Comment:
should _also_ have an `ecto.migrate` in here once we have some data to be inserted. (_it's **fine** like this for now_)